build.sbt

```diff
@@ -487,7 +487,7 @@ lazy val sbtProj = (project in sbtPath).
 lazy val mavenResolverPluginProj = (project in file("sbt-maven-resolver")).
   dependsOn(sbtProj, ivyProj % "test->test").
   settings(
-    baseSettings,
+    testedBaseSettings,
```

Comment:
So the root problem here was that this subproject was genuinely missing library dependencies for tests?

Reply:
No, transitively it should already have specs2. But Ivy evicts the dependency on 0.13.16-SNAPSHOT artefacts (from `version`) in favour of 0.13.16 artefacts (from `sbt.version`).. believe it or not..

Reply:
lol. I am guessing this is yet another Ivy-Maven impedance mismatch since Maven doesn't make Tests public.

Reply:
Also shouldn't we bump the version number if that a cause of so much trouble?

Reply:
in 0.13 everything's Ivy, so it's an impedance mismatch between what Ivy thinks of version strings and which version strings we pick for sbt..